docs/workflow/segments.rst

```diff
@@ -35,19 +35,150 @@ Using this module requires a number of things
 
 The module is then called according to
 
-.. autofunction:: pycbc.workflow.setup_segment_generation
-       :noindex:
+.. autofunction:: pycbc.workflow.get_segments_file
+          :noindex:
 
 -------------------------
 Configuration file setup
 -------------------------
 
+Here we document the necessary parts of a configuration file for this module.
+We'll lay this out in a few blocks. First we'll give an overview of what this might look like in an O3 analysis, to give some background, then we'll give a more comprehensive description of the input format in the `ini` file, and finally we'll explain the full syntax that can be used for individual flags.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Example config file
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. literalinclude:: resources/example_dq.ini
+
+
+Note that this includes both datafind and segment instructions. We'll just
+describe the segment options
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Description of the ini file contents
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are three important segments options that we need to provide:
+
+* `segments-science`: This decides what times will be analysed to produce triggers. All times in this might be used to compute PSDs used in the final results. In previous runs this is normally the science flag with CAT_1 vetoes subtracted from it.
+* `segments-vetoes`: This decides what times will be vetoed when producing final significance. These times *are* analysed but are discarded after combining single-detector triggers together. In previous runs this is CAT_2 vetoes and hardware injections.
+* `segments-veto-definer-url`: As previously, this is the location of the veto definer.
+
+Note that this obeys the usual workflow tagging rules. If you supply `segments-science` in `workflow-segments` it will be valid for all ifos. Or, if you want to supply different values for different ifos (e.g. because the Virgo SCIENCE flag is named differently to L1 and H1) you can use `workflow-segments-ifos`.
```

Comment:
Oh, I hadn't noticed that this difference existed. Is there a link for the usual rules for this case of adding 'ifos' to the tag?

Or do you mean workflow-segments-ifoname where ifoname is h1, v1 etc

Reply:
I mean workflow-segments-ifoname where ifoname is h1, v1 etc, clarifying now.